Dockerfile

```diff
@@ -56,11 +50,8 @@ COPY --from=build /app/demo/stub/threagile.yaml /app/threagile-stub-model.yaml
 
 RUN mkdir /data
 
-RUN chown -R 1000:1000 /app /data
```

Comment:
question: I assume you removed these lines because you were experiencing some errors while running the container.
* What errors were you experiencing?
* What version of docker were you running? (seems like an error in docker-for-linux)

Reply:
I experienced the same error as @cjones-teradici when trying to run threagile via the provided Docker image:

```
gholland@TERVDIU18AZR037 % docker run --rm -it -v "$(pwd)":/app/work threagile/threagile -create-stub-model -output /app/work
panic: open /app/work/threagile-stub-model.yaml: permission denied
goroutine 1 [running]:
main.checkErr(...)
        /app/main.go:663
main.createStubModelFile()
        /app/main.go:3850 +0xab
main.parseCommandlineArgs()
        /app/main.go:3798 +0x26e5
main.main()
        /app/main.go:668 +0x19
```
Note that I have [user namespace remapping setup on my docker daemon](https://docs.docker.com/engine/security/userns-remap/#enable-userns-remap-on-the-daemon).